tests/utils/test_urls.py

```diff
@@ -1,29 +1,29 @@
 from uuid import UUID
 
-from app.utils.urls import get_frontend_download_url, get_direct_file_url
+from app.utils.urls import get_api_download_url, get_direct_file_url
 
 
 SAMPLE_KEY = bytes(range(32))
 # the b64 has one trailing =, that we strip.
 SAMPLE_B64 = 'AAECAwQFBgcICQoLDA0ODxAREhMUFRYXGBkaGxwdHh8'
 
 
-def test_get_frontend_download_url_returns_frontend_url_without_filename(app):
-    assert get_frontend_download_url(
+def test_get_api_download_url_returns_frontend_url_without_filename(app):
+    assert get_api_download_url(
         service_id=UUID(int=0), document_id=UUID(int=1), key=SAMPLE_KEY,
         filename=None
-    ) == 'http://localhost:7001/d/{}/{}?key={}'.format(
+    ) == 'http://localhost:7000/d/{}/{}?key={}'.format(
         'AAAAAAAAAAAAAAAAAAAAAA',
         'AAAAAAAAAAAAAAAAAAAAAQ',
         SAMPLE_B64
     )
 
 
-def test_get_frontend_download_url_returns_frontend_url_with_filename(app):
-    assert get_frontend_download_url(
+def test_get_api_download_url_returns_frontend_url_with_filename(app):
```

Comment:
Maybe we should rename the test as well as in contains a reference to a front-end URL.

Reply:
Good catch!